sdk/monitor/Azure.Monitor.Ingestion/tests/SessionRecords/MonitorIngestionLiveTest/ValidInputFromArrayAsJsonWithSingleBatchWithGzip.json

```diff
@@ -9,24 +9,23 @@
         "Content-Encoding": "gzip",
         "Content-Length": "282",
         "Content-Type": "application/json",
-        "traceparent": "00-d7eab3da7de9074180f69723e6f57835-1fc1853c5c3d0d41-00",
-        "User-Agent": "azsdk-net-Monitor.Ingestion/1.0.0-alpha.20220829.1 (.NET Core 3.1.28; Microsoft Windows 10.0.22000)",
-        "x-ms-client-request-id": "2f53972db13a1676b62485385d80dbc5",
+        "traceparent": "00-db6e6894cd75d747977337ff856c8dd3-87f6403ebe95df43-00",
+        "User-Agent": "azsdk-net-Monitor.Ingestion/1.0.0-alpha.20220916.1 (.NET Core 3.1.29; Microsoft Windows 10.0.22621)",
+        "x-ms-client-request-id": "18efc8d807330dab26e468ec942d4044",
         "x-ms-return-client-request-id": "true"
       },
       "RequestBody": [
         "\u001F\uFFFD\b\u0000\u0000\u0000\u0000\u0000\u0000\n",
         "\uFFFD\uFFFD;\n",
-        "\uFFFD0\u0010E\u047DLm\uFFFD\uFFFDH\uFFFD\u003E]\uFFFD\u0016\uFFFD\u0019\u0015\u0001\uFFFD\u0010\uFFFDV\b\n",
-        "\u0004\uFFFD\uFFFDn\uFFFDH7\uFFFDt\uFFFD\uFFFD\uFFFD\uFFFD\uFFFD\u000F\uFFFD\uFFFDf\uFFFD@H\u0523\uFD5A\uFFFD\uFFFD\uFFFD\uFFFD\uFFFD\uFFFD\u0060\uFFFD\u000B\uFFFDA\u0007c]_\uFFFD\uFFFD\uFFFD\uFFFD\uFFFD?\uFFFD\uFFFD\uFFFDe)\uFFFD\uFFFD\uFFFD\uFFFD\u001C\uFFFD\uFFFD\uFFFDA\uFFFD=u\u0002U\uD969\uDCA9\u012B$S5\uFFFDj\uFFFDjx\uFFFD\uFFFDT\u02EBV\uFFFD\u000E\uFFFD:\uFFFD\uFFFD\u002B3\uFFFD\uFFFD\uFFFD\uFFFDW\uFFFDL\r",
-        "\uFFFD\u001A\uFFFD\uFFFD\u000E\u0000\u0000\u0000\uFFFD\uFFFD"
+        "\uFFFD0\u0010E\u047DLm\u0007\uFFFDH\uFFFD\u003E]\uFFFD\u0016\uFFFD\u0019\u0015\u0001\uFFFD\u0010\uFFFDV\b\n",
```

Comment:
FYI it looks like we will need to re-record these once https://github.com/Azure/azure-sdk-tools/pull/4165 is merged, so probably not worth spending a lot of time on the test recordings before that.

Reply:
I will change the concurrency tests to be LiveOnly for now and add the parameter to change the Upload size as well for testing.

Reply:
Now that I can reduce the size of the batch - we can have recordings for concurrency